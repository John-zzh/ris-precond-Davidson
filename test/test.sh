#!/bin/bash

python ../Davidson.py -x *.xyz -b def2-tzvp -f pbe0 -n 5 -df True -TDA true -o 0 \
-v 3 -chk True -TDDFT_as True -ei 2 -max 10 
