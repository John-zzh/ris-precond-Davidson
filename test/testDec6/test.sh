#!/bin/bash

# python ../Davidson.py -x *.xyz -b def2-svp -f pbe0 -n 5 -df True -TDDFT true -o 0 \
# -v 4 -chk true -TDDFT_as True -ei 2 -max 10 \
# -dscf true \
# -coulomb_ex none \
# -cl_aux_p true -ex_aux_p False \
# -Uc 1 -Ue 1 \
# -TV 1000000 40 -TO 1000000 40 \
# -TDDFT_as_profile true



python ../Davidson.py -x *.xyz -b def2-svp -f pbe0 -n 5 -df True \
-v 3 -chk false -TDDFT_as True \
-cl_aux_p true -ex_aux_p False \
-Uc 1 -Ue 1 \
-TV 1000000 1000000 -TO 1000000 1000000 \
-TDA true  -TDDFT_as_profile true \
-w true -FK true
