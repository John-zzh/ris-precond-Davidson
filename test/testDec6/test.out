RSH_omega = 0
This job can use: 8CPUs
mol.max_memory = 4000
RKS
Density fitting turned on
Molecule built
Calculating SCF Energy...
converged SCF energy = -115.496877084466
SCF Done after 1.87 seconds
after SCF memory used: 0.05 GB
delta_hdiag2 (9, 39)
a_x = 0.25
N_bf = 48
N_atm = 6
n_occ = 9
cl_rest_occ = 9
ex_rest_occ = 9
n_vir = 39
cl_rest_vir =  39
ex_rest_vir =  39
A_size =  351
cl_A_rest_size = 351
ex_A_rest_size = 351
reduced_vir = 39
TDDFT
asigning auxiliary basis set, add p function = True
U = 1.0
asigning auxiliary basis set, add p function = False
U = 1.0
eri2c_cl.shape (12, 12)
eri2c_ex.shape (6, 6)
eri3c.shape (48, 48, 12)
eri3c.shape (48, 48, 6)
curpath /Users/zehaozhou/repo/project/davidson/davidson/testDec6
lib.num_threads() =  1
at beginning memory used: 0.07 GB
xyzfile = methanol.xyz
checkfile = True
dscf = False
method = RKS
functional = pbe0
basis_set = def2-svp
density_fit = True
grid_level = 3
charge = 0
nstates = 5
pytd = False
TDA = False
TDDFT = True
dpolar = False
dpolar_omega = []
spolar = False
sTDA = False
sTDDFT = False
TDDFT_as = True
TDA_as_profile = False
TDDFT_as_profile = False
spolar_as_profile = False
dpolar_as_profile = False
test = True
spectra = False
eta = False
coulomb_aux_add_p = True
exchange_aux_add_p = False
coulomb_U = 1.0
exchange_U = 1.0
Uk_tune = False
truncate_virtual = [1000000.0, 1000000.0]
truncate_occupied = [1000000.0, 1000000.0]
GS_double = False
ip_options = [0]
conv_tolerance = 1e-05
initial_TOL = 0.001
precond_TOL = 0.01
extrainitial = 8
max = 35
memory = 4000
verbose = 3
einsum = pyscf
beta = None
alpha = None
|-------- In-house Developed TDDFT Starts ---------|
====== TDDFT Calculation Starts ======
TDDFT Iterative Initial Guess Done
Finished in 6 steps, 0.04 seconds
final subspace 41
max_norm = 7.14e-04
MVcost     0.0140s 37.82%
GScost     0.0061s 16.47%
subgencost 0.0039s 10.57%
subcost    0.0091s 24.66%
full_cost  0.0015s 4.09%
size_new = 10
initial_energies
[ 7.17785  8.87951  9.06681  9.83952 10.08956]
initial guess done

Iteration 1
beginning of step 0 memory used: 0.08 GB
subspace size: 10
Maximum residual norm: 3.54e-02
unconverged states [0, 1, 2, 3, 4]
TDDFT iterative preconditioner Done
Finished in 7 steps, 0.04 seconds
final subspace 27
max_norm = 7.44e-03
init_cost  0.0014s  3.30%
MVcost     0.0270s  66.05%
GScost     0.0030s  7.25%
subgencost 0.0030s  7.28%
subcost    0.0043s  10.56%
full_cost  0.0012s  2.90%

Iteration 2
beginning of step 1 memory used: 0.08 GB
subspace size: 15
Maximum residual norm: 3.27e-03
unconverged states [0, 1, 2, 3, 4]
TDDFT iterative preconditioner Done
Finished in 6 steps, 0.03 seconds
final subspace 22
max_norm = 9.78e-03
init_cost  0.0009s  3.10%
MVcost     0.0190s  63.07%
GScost     0.0026s  8.50%
subgencost 0.0022s  7.34%
subcost    0.0035s  11.70%
full_cost  0.0010s  3.24%

Iteration 3
beginning of step 2 memory used: 0.08 GB
subspace size: 20
Maximum residual norm: 3.51e-04
unconverged states [0, 1, 2, 3, 4]
TDDFT iterative preconditioner Done
Finished in 6 steps, 0.03 seconds
final subspace 20
max_norm = 9.56e-03
init_cost  0.0009s  3.54%
MVcost     0.0158s  63.04%
GScost     0.0018s  7.11%
subgencost 0.0020s  7.98%
subcost    0.0030s  11.86%
full_cost  0.0009s  3.50%

Iteration 4
beginning of step 3 memory used: 0.08 GB
subspace size: 25
Maximum residual norm: 3.67e-05
unconverged states [2, 4]
TDDFT iterative preconditioner Done
Finished in 7 steps, 0.03 seconds
final subspace 14
max_norm = 4.85e-03
init_cost  0.0006s  1.88%
MVcost     0.0232s  75.46%
GScost     0.0013s  4.25%
subgencost 0.0018s  6.00%
subcost    0.0024s  7.77%
full_cost  0.0007s  2.12%

Iteration 5
beginning of step 4 memory used: 0.08 GB
subspace size: 27
Maximum residual norm: 9.13e-06
TDDFT precedure Done
============= TDDFT Calculation Done ==============
energies:
[ 7.26545  8.92061  9.1469   9.9099  10.13497]
Maximum residual norm = 9.13e-06
Finished in 5 steps, 2.10 seconds
Final subspace size = 27
Initial guess time : 0.04 seconds, 1.83%
Total precondition time: 0.15 seconds, 7.05%
after TDDFT done memory used: 0.08 GB
Initia guess   iter
Preconditioner iter
|-------- In-house Developed TDDFT Ends ----------|
